node/_crypto/crypto_browserify/asn1.js/base/node.js

```diff
@@ -3,7 +3,7 @@
 
 import { Reporter } from "./reporter.js";
 import { DecoderBuffer, EncoderBuffer } from "./buffer.js";
-import { assert, assertEquals } from "../../../../../testing/asserts.ts";
+import { assert, assertEquals } from "../../../../../_util/asserts.ts";
```

Comment:
Let's drop this `assertEquals` usage (it only compares numbers. It should be easily replaced by `assert`). `assertEquals` has fancy formatting feature of value diffs and that's not necessary for internal assertions.